packages/library-base/src/commonMain/kotlin/io/realm/kotlin/Configuration.kt

```diff
@@ -89,6 +90,21 @@ public data class LogConfiguration(
     public val loggers: List<RealmLogger>
 )
 
+/**
+ * Configuration for pre-bundled asset files used as initial state of the realm file.
+ */
+public data class InitialRealmFileConfiguration(
+    /**
+     * Path to the assetFile. This will be interpreted differently depending on the platform. See [SharedBuilder.initialRealmFile] for details.
```

Comment:
```suggestion
     * Path to the realm file. This will be interpreted differently depending on the platform. See [SharedBuilder.initialRealmFile] for details.
```